pkg/compile/merge.go

```diff
@@ -10,7 +10,7 @@ import (
 	"fybrik.io/taxonomy-cli/pkg/slices"
 )
 
-// mergeDefinitions merges the definitions section of multiple documents.
+// mergeDefinitions merges the definitions' section of multiple documents.
```

Comment:
I think the correct way is the previous one, "definitions section".

Reply:
OK, will turn it back.